### PR TITLE
Docs/refresh getting started

### PR DESCRIPTION
**Pull request by Kammerlo**
Added Documentation how to start the service in your local environment. 
Additionally provided an example docker-compose script to set up the needed infrastructure (excl. ledger-sync)

And cleaned up example environment file + added comments for explanation.
This is just a start of adding extra documentation.

### CODE REVIEW

**Comment by Sotatek-DucPhung** [comment]
@Kammerlo i have remove some unsued env vars. Overall LGTM!

**Review by Sotatek-DucPhung — APPROVED** [review]

**Review by nemo83 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,13 +13,43 @@ This repository executes tasks in a periodic sequence to precompute computationa
 
 👉 Check the [Explorer repository](https://github.com/cardano-foundation/cf-explorer) to understand how the microservices work together
 
+## How to run
+Following prerequisites are required to run the application:
+- Java 17
+- Kafka (will be removed soon)
```
Comment (Sotatek-DucPhung) [inline README.md:19]: We can remove Kafka now

File: README.md [hunk 2]
```diff
@@ -13,13 +13,43 @@ This repository executes tasks in a periodic sequence to precompute computationa
...
+- Redis 
+- Postgres (ledger-sync and explorer database)
+- Ledger-sync (Ledgersync + Aggregate app + Postgres)
+
+An example docker-compose script is provided in the `example-infra` directory. 
+(This script only sets up Kafka, Redis and Postgres, for a ledger-sync setup, please refer to the [ledger-sync repository](https://github.com/cardano-foundation/cf-ledger-sync))
```
Comment (Sotatek-DucPhung) [inline README.md:25]: same here, unused Kafka

File: README.md [hunk 3]
```diff
@@ -13,13 +13,43 @@ This repository executes tasks in a periodic sequence to precompute computationa
...
+
+This can be used right away with the provided `.env.example` and adjusted as needed.
+
+To start the needed Infrastructure, run:
+```shell
+# Start Kafka, Redis and Postgres
```
Comment (Sotatek-DucPhung) [inline README.md:31]: same here, unused Kafka

**Pull request merged**